dp/generate_climos.py

```diff
@@ -442,11 +442,14 @@ def convert_flux_var_units(input_file, climo_data):
         flux_variable = input_file.variables[flux_var]
         units = Unit.from_udunits_str(flux_variable.units)
 
-        if units in [Unit('kg / m**2 / s'), Unit('mm / s')]:
-            logger.info("Converting {} variable to units mm/day".format(flux_var))
+        if units in [Unit('kg / m**2 / s'), Unit('mm / s'), Unit('g / cm**2 / s'), Unit('cm / s')]:
+            if units in [Unit('kg / m**2 / s'), Unit('mm / s')]:
```

Comment:
Do you need this conditional? It only changes one thing... the output unit in the log message. But you already *have* the string for the output unit string below: `(units * Unit('s / day')).to_udunits_str()`. Save that here, then use it both for your log message and for your attributes. Then the conditional can go away.

See ["If Statements are a Code Smell"](https://us.pycon.org/2020/schedule/presentation/87/).